src/sandstorm/grain.capnp

```diff
@@ -633,7 +633,20 @@ interface SessionContext {
   #   saveLabel: A string petname to give this label. This will be displayed to the user as the name
   #          for this capability.
   #
-  # (eg. window.parent.postMessage({powerboxRequest: {rpcId: myRpcId, query: [{}]}}, "*")
+  # e.g.:
+  #   window.parent.postMessage({
+  #     powerboxRequest: {
+  #       rpcId: myRpcId,
+  #       query: [
+  #         {
+  #           tags: [
+  #             { id: "10145426045091321204" },
```

Comment:
I think that in any case the `descriptor` field in the `postMessage()` call should be a valid node-capnp value. 

If we don't accept base-16 strings, I think we should at least provide some javascript that apps can use to make the conversion themselves.